gdk4/Gir.toml

```diff
@@ -566,6 +566,14 @@ name = "Gdk.MemoryTexture"
 status = "generate"
 concurrency = "send+sync"
 
+[[object]]
+name = "Gdk.MemoryTextureBuilder"
+status = "generate"
+concurrency = "send+sync"
+    [[object.function]]
+    name = "set_update_region"
```

Comment:
There are plenty of examples using const = true

Reply:
I have no idea what exactly it does but I have added it.